tests/parsers/aws_elb_access.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""Tests for Apache access log parser."""
+
+import unittest
+
+from plaso.parsers import aws_elb_access
+from tests.parsers import test_lib
+
+
+class AWSELBUnitTest(test_lib.ParserTestCase):
+  """Tests for AWS ELB access log parser."""
+
+  def testParse(self):
+    """Tests the Parse function."""
+    parser = aws_elb_access.AWSELBParser()
+    storage_writer = self._ParseFile(['aws_elb_access.log'], parser)
+
+    # Test number of events and wrarnings
+    self.assertEqual(storage_writer.number_of_events, 10)
```

Comment:
This interface will be deprecated soon and has been updated recently
PTAL at how this is tested in the current version of the codebase

Reply:
I edited the tests to utilize the GetNumberOfAttributeContainers() method